packages/shared-types/src/lib/authorization.types.ts

```diff
@@ -0,0 +1,25 @@
+import type { ConfigOptions } from './config.types.js';
+
+/**
+ * Define the options for the authorization URL
+ * @param clientId The client ID of the application
+ * @param redirectUri The redirect URI of the application
+ * @param responseType The response type of the authorization request
+ * @param scope The scope of the authorization request
+ */
+export type ResponseType = 'code' | 'token';
+export interface GetAuthorizationUrlOptions extends ConfigOptions {
```

Comment:
Is this the best place for this interface? It's specific to the getAuthorizeUrl function so maybe it should live with it in authorization.types.ts in the effects package.

Reply:
Yeah, it may be able to live there, I was moving a lot of stuff so I was trying to lower things down in the tree and revisit them once it was all working

Reply:
Good question, @ancheetah. I'd say that types should live as close to the code it defines, unless there's a risk of circular dependencies, or if it defines things that are not directly related to another package.